### PR TITLE
added definitions for darkroomjs

### PR DESCRIPTION
**Pull request by stepancar**
added definitions for https://github.com/MattKetmo/darkroomjs

### CODE REVIEW

**Comment by stepancar** [comment]
@vvakame, merge it please

**Comment by vvakame** [comment]
*darkroomjs/darkroomjs.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/darkroomjs
  * http://bower.io/search/?q=darkroomjs
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (darkroomjs/darkroomjs-tests.ts or others)
* [ ] pass the Travis-CI test?

**Comment by vvakame** [comment]
@stepancar thank you for your contributions.
please rename darkroomjs/darkroomjs.d.ts to darkroom/darkroom.d.ts

File: darkroomjs/darkroomjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        crop: any;
+        save: any;
+    }
+    sourceImage: fabric.IImage;
+    /**
+    * add event listener for fabric events
+    */
+    addEventListener: (eventName: string, callBack:()=>any)=>any
+}
+interface Options {
```
Comment (vvakame) [inline darkroomjs/darkroomjs.d.ts]: `Options` is very popular identifier. you should use ghost module pattern. http://definitelytyped.org/guides/best-practices.html#ghost-modules

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**